content/grants/app-engram.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Engram: a collaborative knowledge managment app"
+date = "2022-03-25"
+
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Engram is a knowledge management application that allows individuals or teams to build rich note documents."
+reward = "$70,000 in WSTR"
+assignee = ""
+id = ""
+completed = false
+work_request_link = ""
```

Comment:
@mikeosborne can you add a work request link here please? Then this is good to go.

Reply:
It is needed, as it's what allows people to apply for the bounty. Mike needs to add it though.